tests/project.bats

```diff
@@ -27,4 +27,8 @@ setup() {
 	assert_output --partial "generated by dagger"
 
 	test ! -f ./cue.mod/pkg/.gitignore
+
+	"$DAGGER" project info
```

Comment:
You will have to add `run ` before `"$DAGGER`, e.g.

```
run "$DAGGER" project info
assert_success
assert_output ...
```

`assert_*` functions are helpers that work with `run` (it's asserting the last run)

Reply:
Ahh ok, updated with change.